st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -514,6 +524,51 @@ def _get_pkey_object(self, key_material):
 
         raise paramiko.ssh_exception.SSHException(msg)
 
+    def _bastion_connect(self):
```

Comment:
I had thought about this, but the solution I came up with wasn't very elegant- it involved adding boolean parameters to a shared `_connect` method that `connect` would call twice, depending on whether a bounced connection was required. I'll take a stab at it but I don't think it would help with code readability.